.github/labeler.yml

```diff
@@ -43,3 +43,11 @@ documentation:
 "QL-for-QL":
   - ql/**/*
   - .github/workflows/ql-for-ql*
+
+# since these are all shared files that needs to be synced, I just picked _one_ version
```

Comment:
Grammar s/needs/need/ and getting rid of the "I", which is unclear unless you look at the commit history.
```suggestion
# Since these are all shared files that need to be synced, just pick _one_ copy of each.
```

Reply:
thanks